CHANGELOG.md

```diff
@@ -5,6 +5,36 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.9.4] - 2024-10-10
+
+- revert: "feat: add local langfuse tracing option (#106)"
```

Comment:
quotes around this but not others

Reply:
this is the same as the commit msg since we reverted a PR